installer/dev/tracelogging.h

```diff
@@ -4,10 +4,9 @@
 #pragma once
 
 #include "pch.h"
-#include "MicrosoftTelemetry.h"
 
-#if __has_include(<WindowsAppSDK-VersionInfo.h>)
-#include <WindowsAppSDK-VersionInfo.h>
+#if __has_include("WindowsAppSDK-VersionInfo.h")
```

Comment:
Why "" instead of <>?

"" means "first look relative to current/project directory" but this file lives in the root, not under root\installer, right? Found via $(AdditionalIncludeDirectories)

Reply:
No. CreateInstaller stage doesn't share the same file system as the Aggregates. Hence, I am having to extract the WindowsAppSDK  nuget package build artifact from Aggregates stage and copy the version info header file to current directory.